docs/ContainerRuntimeConfigDesign.md

```diff
@@ -56,7 +56,23 @@ ContainerRuntimeConfig
  [ContainerRuntimeConfigurationSpec](Will have to add this in this repo)
 ```
 
-## Example 
+## VALIDATION
+
+It's importent to note that, since the fields of the ContainerRuntimeConfig are directly read by the upstream kubernetes golang client, the validation of those values is handled directly by that golang client which is outside of the controller for ContainerRuntimeConfig. Please ensure the valid values are used for those fields as invalid values may render cluster nodes unusable.
```

Comment:
```suggestion
It's important to note that, since the fields of the ContainerRuntimeConfig are directly read by the upstream kubernetes golang client, the validation of those values is handled directly by that golang client which is outside of the controller for ContainerRuntimeConfig. Please ensure the valid values are used for those fields as invalid values may render cluster nodes unusable.
```

Reply:
Also are there an upstream docs to link to on this upstream validation?